src/pages/registration/RegistrationForm.tsx

```diff
@@ -125,6 +147,14 @@ export const RegistrationForm = ({ identifier }: RegistrationFormProps) => {
           </Form>
         )}
       </Formik>
+      <ConfirmDialog
+        open={showNviWarning}
+        title={t('registration.nvi_warning.registration_is_included_in_nvi')}
+        onAccept={() => setHasAcceptedNviWarning(true)}
+        onCancel={() => (history.length > 1 ? history.goBack() : history.push(UrlPathTemplate.Home))}>
+        <Typography paragraph>{t('registration.nvi_warning.reset_nvi_warning')}</Typography>
+        <Typography>{t('registration.nvi_warning.continue_editing_registration')}</Typography>
```

Comment:
Slå sammen disse til en streng med `\n\n` kanskje?

Reply:
samtidig kan man argumentere for at med er mest riktig mtp a11y å ha to `<p>` tags her da 🤔

Reply:
lar det være to innslag